x/cardchain/client/cli/query_q_cards.go

```diff
@@ -31,6 +33,15 @@ func CmdQCards() *cobra.Command {
 			reqKeywordsContains := args[6]
 			reqNotesContains := args[7]
 
+			switch args[8]{
+			case "yes":
+				reqOnlyStarterCard = true
+			case "no":
+				reqOnlyStarterCard = false
+			default:
+				return fmt.Errorf("arg 'only-startercard' has to be either yes or no but not %s", args[8])
```

Comment:
instead of yes and no, should we not use true or false?

Reply:
no